recipes/vcboost/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0" %} 
+
+package: 
+  name: vcboost 
+  version: {{ version }} 
+  
+source: 
+  url: https://github.com/oranges7/VCboost/archive/refs/heads/main.zip
+  md5: 43a113509e361c9f262115a74add49e8
+  sha1: 97070bb1487adc6e1b14642d99f1cfba99dd2ba7
+  sha256: e46d76702e92aba318d50169b64188f62aea5d5839d610cc196aadd8ac199073 
+  
+build: 
+  noarch: generic 
+  number: 0 
+  run_exports:
+  - vcboost 1.0.0  
+test: 
+  source_files:
+    - vcboost.sh
+    - script/predict/run_predict.sh
+  commands:
+    - sh vcboost.sh -h
+requirements: 
+  host: 
+    - python 
+  run: 
+    - python ==3.9.0 
```

Comment:
please do not pin your dependencies that strict, only use pins when actually necessary